js/desktopCapturer/screen-picker.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Screen Picker</title>
+    <style>
+
+        body {
+            margin: 0 auto;
+            font-family: "Helvetica Neue", sans-serif;
+            overflow: hidden;
+        }
+
+        .window-border {
+            border: 2px solid #444;
+        }
+
+        .content {
+            margin: 0 auto;
+            background: #F9F9FA;
+            text-align: center;
+            height: 100%;
+            overflow: hidden;
+        }
+
+        .custom-window-title {
+            display: flex;
+            flex-flow: row nowrap;
+            align-items: center;
+            justify-content: space-between;
+            padding: 16px 16px 30px;
+            font-size: 1.2rem;
+            line-height: 1.3;
+            text-align: left;
+        }
+
+        #x-button {
+            width: 20px;
+            color: rgba(221, 221, 221, 1);
+        }
+
+        span {
+            font-style: normal;
+            margin: 0 0 0 4px;
+            font-size: 1.2rem;
+            min-height: 13px;
+        }
+
+        input {
+            display: none;
```

Comment:
Why is this required?

Reply:
Input type for the tabs that I'm using is `radio` so in order to hide the radio button setting `display: none`